src/api/public/apidocs/OBS-v2.9.yaml

```diff
@@ -0,0 +1,114 @@
+openapi: 3.0.0
+servers: []
+info:
+  description: Open Build Service API
+  version: '2.9'
+  title: Open Build Service API
+  contact:
+    url: 'https://openbuildservice.org/'
+    name: Open Build Service
+  license:
+    url: 'https://github.com/openSUSE/open-build-service/blob/master/COPYING'
+    name: GNU General Public License v2.0
+paths:
+  /about:
+    get:
+      summary: Get information about API.
+      description: Get generic information about the API.
+      responses:
+        '200':
+          description: OK. The request has succeeded.
+          content:
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/about'
+  /group:
+    get:
+      summary: List available groups.
+      description: List available groups.
+      responses:
+        '200':
+          description: OK. The request has succeeded.
+          content:
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/group_list'
+  /group/{group_title}:
+    get:
+      summary: Read group data.
+      description: Read group data.
+      parameters:
+        - in: path
+          name: group_title
+          schema:
+            type: string
+          required: true
+          description: Group title
+          example: group-admin
+      responses:
+        '200':
+          description: OK. The request has succeeded.
+          content:
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/group'
+components:
+  schemas:
+    about:
```

Comment:
As your schema grows, having all definitions in one file can get annoying.
You can also include external files, like so:

https://github.com/SUSE/happy-customer/blob/master/glue/public/api_docs/swagger/swagger.yaml#L141

Reply:
The link gives 404. Is it a private repo?

Reply:
The repo is private, but the referenced Swagger file is public:
https://scc.suse.com/connect/v4/swagger.yaml